apps/openmw/mwworld/store.cpp

```diff
@@ -1057,44 +1057,23 @@ namespace MWWorld
     void Store<ESM::Static>::setUp()
     {
         // Load default marker definitions, if game files do not have them for some reason
-        const ESM::Static *ptr = search("divinemarker");
-        if (ptr == 0)
+        std::pair<std::string, std::string> markers[] = {
+            std::make_pair("divinemarker", "marker_divine.nif"),
+            std::make_pair("doormarker", "marker_arrow.nif"),
+            std::make_pair("northmarker", "marker_north.nif"),
+            std::make_pair("templemarker", "marker_temple.nif"),
+            std::make_pair("travelmarker", "marker_travel.nif")
+        };
+
+        for (std::pair<std::string, std::string> marker : markers)
```

Comment:
This will copy both strings of each element since it's taken by value. You can do `const auto &marker : markers` instead to take a reference to each pair.

Reply:
Done.